lib/views/home/home_view.dart

```diff
@@ -41,7 +41,7 @@ class _HomeViewState extends ConsumerState<HomeView>
             children: [
               Expanded(
                 child: RefreshIndicator(
-                  onRefresh: _onRefresh,
+                  onRefresh: onRefresh,
```

Comment:
This should be private shouldn't it?

Reply:
Yeah this should be private damn it remove underscore from all the places lol let me fix it real quick